charts/kyverno/values.yaml

```diff
@@ -65,6 +65,11 @@ apiVersionOverride:
   # determine the api version automatically.
   podDisruptionBudget: ~
 
+rbac:
+
+  # Aggregate ClusterRoles to Kubernetes default user-facing roles. For more information, see [User-facing roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)
+    aggregateClusterRoles: true
```

Comment:
@allanp35 - the indentation issue to fix :)

Reply:
@realshuting - is it good now? ;)